src/WooCommerce.php

```diff
@@ -223,7 +223,14 @@ public static function woocommerce_process_product_meta($post_id) {
     foreach ($custom_fields as $field) {
       if (isset($_POST[$field])) {
         if (!is_array($_POST[$field]) && $_POST[$field]) {
-          update_post_meta($post_id, $field, $_POST[$field]);
+          if ($field !== '_' . Plugin::PREFIX . '_gtin') {
+            update_post_meta($post_id, $field, $_POST[$field]);
+          } else if (!self::is_existing_gtin($post_id, $_POST[$field])) {
```

Comment:
We are following [Drupal Code Standards for control structures](https://www.drupal.org/docs/develop/standards/coding-standards#controlstruct). Therefore, the `else if` should be on a new line.